### PR TITLE
Cascading "New Tab" / "New Window" menu

### PR DESCRIPTION
**Pull request by peske**
**Merge https://github.com/felixse/FluentTerminal/pull/594 first!** This one builds upon it.

As I've promised, I've created cascading "New Tab" / "New Window" menu (although I don't like it :))

image

Good thing is that the new cascading menu now can be turned on/off by simply changing `MainViewModel.TabWindowCascadingMenu` constant. Maybe we can add this constant to settings?

### CODE REVIEW

**Comment by felixse** [comment]
Configurable sounds good, thank you 😃

File: FluentTerminal.App.ViewModels/MainViewModel.cs [hunk 1]
```diff
@@ -304,8 +341,8 @@ private async Task AddSshProfileAsync(NewTerminalLocation location)
 
         #region Quick launch profile
 
-        private Task AddQuickLaunchProfileAsync() =>
-            AddQuickLaunchProfileAsync(_settingsService.GetApplicationSettings().NewTerminalLocation);
+        //private Task AddQuickLaunchProfileAsync() =>
```
Comment (felixse) [inline FluentTerminal.App.ViewModels/MainViewModel.cs]: can those be deleted?

Comment (peske) [reply FluentTerminal.App.ViewModels/MainViewModel.cs]: Deleted.

**Comment by peske** [comment]
Cascading menu is now configurable:

image

File: FluentTerminal.App/Strings/en/Resources.resw [hunk 2]
```diff
@@ -1074,4 +1068,25 @@ Fuzzy</comment>
...
+    <value>Launches a session in a new tab.</value>
+  </data>
+  <data name="MenuItem_NewWindow_Description" xml:space="preserve">
+    <value>Launches a session in a new window.</value>
+  </data>
+  <data name="AppMenuType.Text" xml:space="preserve">
+    <value>Application menu type</value>
+  </data>
+  <data name="AppMenuType_Flat.Content" xml:space="preserve">
+    <value>Flat, honoring &quot;Open new terminals as&quot; settings</value>
```
Comment (mjs) [inline FluentTerminal.App/Strings/en/Resources.resw]: setting

File: FluentTerminal.App/Views/SettingsPages/GeneralSettings.xaml [hunk 3]
```diff
@@ -88,7 +94,24 @@
...
+                </StackPanel>
+
+                <StackPanel Margin="{StaticResource ItemMargin}">
+                    <TextBlock
+                        x:Uid="AppMenuType"
+                        Margin="0,0,0,8"
+                        Text="Application menu type" />
+                    <RadioButton
+                        x:Uid="AppMenuType_Flat"
+                        Content="Flat, honoring 'Open new terminals as' settings"
```
Comment (mjs) [inline FluentTerminal.App/Views/SettingsPages/GeneralSettings.xaml]: setting

File: FluentTerminal.App/Strings/en/Resources.resw (hunk 2 above)
Comment (peske) [reply FluentTerminal.App/Strings/en/Resources.resw]: Fixed.

File: FluentTerminal.App/Views/SettingsPages/GeneralSettings.xaml (hunk 3 above)
Comment (peske) [reply FluentTerminal.App/Views/SettingsPages/GeneralSettings.xaml]: Fixed.

**Comment by peske** [comment]
@felixse, current default is _flat_ menu. If you want to change default to _cascading_ - feel free to do it (you know what needs to be done).

**Pull request merged**